mindboggle/mindboggle123

```diff
@@ -87,6 +87,9 @@ adv_args.add_argument("--fs_openmp", dest="openmp",
 adv_args.add_argument("--fs_T2image", dest="T2image",
                       type=str,
                       help="Optional T2 image to use with FreeSurfer")
+adv_args.add_argument("--fs_3T", dest="T3",
```

Comment:
Hey @satra & @binarybottle, sorry for the delayed response (pre OHBM struggle). 
Yes, I'll work on it the next couple of days and update it ASAP.